packages/react-components/src/theming/icons.tsx

```diff
@@ -246,8 +243,7 @@ export const DEFAULT_COMPONENT_ICONS = {
   SendBoxSend: <Send20Regular />,
   SendBoxSendHovered: <Send20Filled />,
   VideoTileMicOff: <MicOff16Filled />,
-  /* @conditional-compile-remove(dialpad) */ /* @conditional-compile-remove(PSTN-calls) */
-  BackSpace: <Backspace20Regular />,
```

Comment:
Given this is still in beta, consider instead updating this icon name to `DialpadBackspace`

Reply:
Updated